### ISSUE TITLE
vagrant-disksize plugin doesn't trigger in 2.2.13

### ISSUE THREAD

**Issue by makegofast** [hashicorp/vagrant#12035]
We use vagrant-disksize plugin to automatically resize the main disk on provision. This has been working fine up until we upgraded to 2.2.12. In 2.2.12 it seems it doesn't even try to call the vagrant-disksize plugin. By downgrading to 2.2.10 it is once again running.

<will add logs shortly>

**Comment by isuftin** [comment]
Seeing the same issue with vagrant-certificates plugin.

Thought it was just me.

**Comment by makegofast** [comment]
2.2.10:
```
Vagrant 2.2.10
Deez-Nuts:nyx-development-environment jwebster$ vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Importing base box 'ubuntu/bionic64'...
==> default: Matching MAC address for NAT networking...
==> default: Checking if box 'ubuntu/bionic64' version '20200922.0.0' is up to date...
==> default: Setting the name of the VM: ############
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
==> default: Forwarding ports...
    default: 3306 (guest) => 3306 (host) (adapter 1)
    default: 5000 (guest) => 5000 (host) (adapter 1)
    default: 5001 (guest) => 5001 (host) (adapter 1)
    default: 5002 (guest) => 5002 (host) (adapter 1)
    default: 6000 (guest) => 6000 (host) (adapter 1)
    default: 6001 (guest) => 6001 (host) (adapter 1)
    default: 6002 (guest) => 6002 (host) (adapter 1)
    default: 8002 (guest) => 8002 (host) (adapter 1)
    default: 8443 (guest) => 8443 (host) (adapter 1)
    default: 9000 (guest) => 9000 (host) (adapter 1)
    default: 9001 (guest) => 9001 (host) (adapter 1)
    default: 9002 (guest) => 9002 (host) (adapter 1)
    default: 27017 (guest) => 27017 (host) (adapter 1)
    default: 22 (guest) => 2222 (host) (adapter 1)
==> default: Running 'pre-boot' VM customizations...
==> default: Resized disk: old 10240 MB, req 51200 MB, new 51200 MB
==> default: You may need to resize the filesystem from within the guest.
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 127.0.0.1:2222
    default: SSH username: vagrant
    default: SSH auth method: private key
    default:
    default: Vagrant insecure key detected. Vagrant will automatically replace
...
```

2.2.12:
```
Deez-Nuts:nyx-development-environment jwebster$ vagrant --version
Vagrant 2.2.13
Deez-Nuts:nyx-development-environment jwebster$ vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Importing base box 'ubuntu/bionic64'...
==> default: Matching MAC address for NAT networking...
==> default: Checking if box 'ubuntu/bionic64' version '20201109.0.0' is up to date...
==> default: Setting the name of the VM: dev-nyx-env
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
==> default: Forwarding ports...
    default: 3306 (guest) => 3306 (host) (adapter 1)
    default: 5000 (guest) => 5000 (host) (adapter 1)
    default: 5001 (guest) => 5001 (host) (adapter 1)
    default: 5002 (guest) => 5002 (host) (adapter 1)
    default: 6000 (guest) => 6000 (host) (adapter 1)
    default: 6001 (guest) => 6001 (host) (adapter 1)
    default: 6002 (guest) => 6002 (host) (adapter 1)
    default: 8002 (guest) => 8002 (host) (adapter 1)
    default: 8443 (guest) => 8443 (host) (adapter 1)
    default: 9000 (guest) => 9000 (host) (adapter 1)
    default: 9001 (guest) => 9001 (host) (adapter 1)
    default: 9002 (guest) => 9002 (host) (adapter 1)
    default: 27017 (guest) => 27017 (host) (adapter 1)
    default: 22 (guest) => 2222 (host) (adapter 1)
==> default: Running 'pre-boot' VM customizations...
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 127.0.0.1:2222
    default: SSH username: vagrant
    default: SSH auth method: private key
...
```

**Comment by gael-ian** [comment]
Same issue with vagrant-bindfs plugin. Plugin configuration class is loaded and methods can be used in Vagrantfile but no real action is triggered.

**Comment by chrisroberts** [comment]
Test packages with fix applied: https://github.com/hashicorp/vagrant-installers/releases/tag/v2.2.14.dev%2B157db399

**Comment by gael-ian** [comment]
> Test packages with fix applied: https://github.com/hashicorp/vagrant-installers/releases/tag/v2.2.14.dev%2B157db399

Works perfectly :+1:   
Thanks!

**Comment by fnordfish** [comment]
@chrisroberts 157db399 seems to fix vagrant-vbguest as well. Is this probably going to be included in 2.2.14? If so, I might ship a vagrant-vbguest upgrade using a different hook for 2.2.11..2.2.13.

**Comment by oxyc** [comment]
I can confirm this fixes the same issue with `vagrant-hostmanager` as well.

**Comment by chrisroberts** [comment]
@fnordfish Yes, the fix is going to be included in 2.2.14 which we are looking to get shipped very shortly.
